models/git/branches_test.go

```diff
@@ -2,12 +2,14 @@
 // Use of this source code is governed by a MIT-style
 // license that can be found in the LICENSE file.
 
-package models
+package git_test
```

Comment:
Because the test may depends on some sub packages which depends on `git`. Since all testing functions are public. A standalone test package is more flexible.